src/main/java/org/apache/sysds/api/DMLScript.java

```diff
@@ -83,35 +83,36 @@
 
 public class DMLScript 
 {
-	private static ExecMode   EXEC_MODE          = DMLOptions.defaultOptions.execMode;           // the execution mode
```

Comment:
Hi, you can always create a new PR with formatting changes!

Reply:
Should I create a separate PR for the formatting changes next time?

Reply:
Generally, changing formatting is a bit confusing for the reviewers, and difficult to check for accidental typos. I think it's okay for this PR. :)

> An example: https://github.com/apache/systemds/commit/bf4ba16b9aaa9afee20a3f1c03b0ff49c5346a9d

Reply:
Alright, thank you for the feedback. I will try to isolate the formatting changes from the actual PR next time. :)